src/WebJobs.Extensions.NotificationHubs/NotificationHubAttribute.cs

```diff
@@ -39,12 +39,14 @@ public sealed class NotificationHubAttribute : Attribute
         /// Optional. A string value indicating the app setting to use as the Notification Hubs connection
         /// string, if different than the one specified in the <see cref="NotificationHubsConfiguration"/>.
         /// </summary>
+        [AutoResolve(AllowTokens = false)]
         public string ConnectionStringSetting { get; set; }
 
         /// <summary>
         /// Optional. The Notification Hub Name to use, if different than the one specified in the
         /// <see cref="NotificationHubsConfiguration"/>.
         /// </summary>
+        [AutoResolve(AllowTokens = false)]
```

Comment:
Currently, UI does not support picking NotificationHub. But eventually we need to improve this. NotificationHubName is different from QueueName as it needs to be pre-configured before using it.